pvacseq/lib/pipeline.py

```diff
@@ -289,10 +286,9 @@ def execute(self):
             os.unlink(symlink)
 
 
-        print("\n")
-        print("Done: pvacseq has completed. File %s contains list of filtered putative neoantigens" % self.final_path())
-        print("We recommend appending coverage information and running `pvacseq coverage_filter` to filter based on sequencing coverage information")
-        sys.stdout.flush()
+        status_message("\n")
+        status_message("Done: pvacseq has completed. File %s contains list of filtered putative neoantigens" % self.final_path())
+        status_message("We recommend appending coverage information and running `pvacseq coverage_filter` to filter based on sequencing coverage information")
```

Comment:
Would it make sense to combine these into a single `status_message()` call?  It seems a little excessive to `flush` three times, and they look like a single logical message.

(Or maybe `status_message` optionally accepts multiple strings?)